ObjectiveGit/GTSignature.h

```diff
@@ -42,10 +42,10 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly, copy, nullable) NSString *email;
 
 /// The time when the action happened.
-@property (nonatomic, readonly, strong) NSDate *time;
+@property (nonatomic, readonly, strong, nullable) NSDate *time;
```

Comment:
The other properties are nullable, so this is consistent.

On the other hand, they all return nil if `git_signature` is nil, and it's not nullable, and it doesn't look like it should ever be null.

But since I'm making this consistent with the other properties, I'll leave that as a separate issue.